en/frames_multicopter/holybro_qav250_pixhawk4_mini.md

```diff
@@ -0,0 +1,291 @@
+Velcro battery strap          |  1
+Foam for battery              |  1
+Landing pads                  |  4
+
+![QAV250 components for frame](../../assets/airframes/multicopter/qav250_holybro_pixhawk4_mini/frame_components.jpg)
+
+
+### Electronics
+
+Description | Quantity
+--- | ---
+Motors - DR2205 KV2300 |  4
+Fully assembled Power Management Board with ESCs |  4
+Holybro power module   |  1
+Fr-sky D4R-II receiver |  1
+Pixhawk 4 mini         |  1
+Holybro GPS Neo-M8N    |  1
+Holybro [Micro FPV Radio telemetry set](http://www.holybro.com/product/17) | 1
+Battery lumenier 1300 mAh 4S 14.8V |  1
+Vtx 5.8gHz             |  1
+FPV camera (Complete Kit - only) |  1
+
+The image below shows both frame and electronic components.
+
+![QAV250 Frame/Pixhawk 4 Mini Electronics before assembly](../../assets/airframes/multicopter/qav250_holybro_pixhawk4_mini/frame_and_electronics_components.jpg)
+
+
+
+## Assembly
+
+Estimated time to assemble frame is 2 hours and 1.5 hours installing the autopilot and configuring the airframe in *QGroundControl*.
+
+### Tools needed
+
+The following tools are used in this assembly:
+
+- 2.0mm Hex screwdriver
+- 3mm Phillips screwdriver
+- Wire cutters
+- Precision tweezers
+
+![Tools required for assembling QAV250](../../assets/airframes/multicopter/qav250_holybro_pixhawk4_mini/assembly_tools.jpg)
+
+
+### Frame assembly 
+
+1. Attach arms to the button plate with the 15mm screws as shown:
+
+   ![QAV250 Add arms to button plate](../../assets/airframes/multicopter/qav250_holybro_pixhawk4_mini/1_button_plate_add_arms.jpg)
+1. Put the short plate over the arms
+
+   ![QAV250 Add short plate over arms](../../assets/airframes/multicopter/qav250_holybro_pixhawk4_mini/2_short_plate_over_arms.jpg)
+1. Put the nuts on the 15mm screws (shown next step)
+1. Insert the plastic screws into the indicated holes (note that this part of the frame faces down when the vehicle is complete).
+   ![QAV250 Add nuts to 15mm screws and put  plastic nuts in holes](../../assets/airframes/multicopter/qav250_holybro_pixhawk4_mini/3_nuts_screws_holes.jpg)
+1. Add the plastic nuts to the screws (turn over, as shown)
+   ![QAV250 Plastic nuts onto screws](../../assets/airframes/multicopter/qav250_holybro_pixhawk4_mini/4_plastic_nuts_on_screws.jpg)
+1. Lower the power module over the plastic screws and then add the plastics standoffs
+   ![QAV250 Add power module and standoffs](../../assets/airframes/multicopter/qav250_holybro_pixhawk4_mini/5_power_module_on_screws.jpg)
+1. Put the flight controller plate on the standoffs (over the power module)
+   ![QAV250 Add flight controller plate](../../assets/airframes/multicopter/qav250_holybro_pixhawk4_mini/6_flight_controller_plate.jpg)
+1. Attach the motors. The motors have an arrow indicating the direction of rotation.
+   ![QAV250 Add motors](../../assets/airframes/multicopter/qav250_holybro_pixhawk4_mini/7_motors.jpg)
+1. Paste the damping foam on the pixhawk and paste on the flight controller plate
```

Comment:
FYI mine did not come with damping foam, but double-sided tape. Works fine though.

Reply:
The pixhawk came with 4 pieces of foam - would that be best thing to use?
![image](https://user-images.githubusercontent.com/5368500/51976661-5a1f1380-24d9-11e9-904f-2373b7b3cab5.png)

Reply:
I might have overlooked that. But since the Pixhawk has internal IMU damping, the double-sided tape is enough. Any additional foam might even be worse - we'll have to try if the parameters still work.

Reply:
Awesome. By tape, you just mean this stuff right?:
![image](https://user-images.githubusercontent.com/5368500/52022389-d0109280-254c-11e9-9f8c-2a50cbba5c41.png)

Reply:
Exactly.